src/main/java/com/redhat/cloud/notifications/GwResource.java

```diff
@@ -137,4 +143,12 @@ public static String serializeAction(Action action) throws IOException {
                 .build();
         return Message.of(payload).addMetadata(metadata);
     }
+
+    boolean isBundleApplicationEventTypeTripleValid(String bundle, String application, String eventType) {
+        try {
+            return restValidationClient.isBundleApplicationEventTypeTripleValid(bundle, application, eventType).getStatus() == 200;
```

Comment:
Should we retry the call in case of failure?

Reply:
If it is a remote / network failure we should give back a "retry later" code and make the caller suffer.

On top: we _could_ cache the result of those calls for 5mins. b/a/et don't constantly change.

Reply:
Is it really necessary to cache it? I actually thought to start as simple as possible, see if it's doing what we want it to do and add caches later if it's necessary.

Reply:
I said _could_ not _must_

Reply:
Added a retry